bin/setup

```diff
@@ -1,25 +1,28 @@
 #!/usr/bin/env ruby
-require "fileutils"
+require 'fileutils'
```

Comment:
Have you linted these munchkins? I'd probably preserve them 100% as they come + our config if so

Reply:
There were some small changes in the file, and Rubocop complains now, that's why I had to make those those changes.